code/modules/vore/resizing/resize_vr.dm

```diff
@@ -156,37 +157,40 @@ var/const/RESIZE_A_SMALLTINY = (RESIZE_SMALL + RESIZE_TINY) / 2
  * Called from /mob/living/Bump() in the 'brohugs all around' section.
  * @return false if normal code should continue, true to prevent normal code.
  */
-/mob/living/proc/handle_micro_bump_helping(var/mob/living/tmob)
+/mob/living/proc/handle_micro_bump_helping(mob/living/tmob)
 	//Both small! Go ahead and go.
-	if(src.get_effective_size() <= RESIZE_A_SMALLTINY && tmob.get_effective_size() <= RESIZE_A_SMALLTINY)
+	if(get_effective_size() <= RESIZE_A_SMALLTINY && tmob.get_effective_size() <= RESIZE_A_SMALLTINY)
 		return TRUE
 
 	//Worthy of doing messages at all
 	if(abs(get_effective_size() - tmob.get_effective_size()) >= 0.50)
+		var/src_message = null
+		var/tmob_message = null
 
 		//Smaller person being stepped onto
 		if(get_effective_size() > tmob.get_effective_size() && ishuman(src))
+			src_message = "You carefully step over [tmob]."
+			tmob_message = "[src] steps over you carefully!"
 			var/mob/living/carbon/human/H = src
 			if(H.flying)
 				return TRUE //Silently pass without a message.
 			if(isTaurTail(H.tail_style))
 				var/datum/sprite_accessory/tail/taur/tail = H.tail_style
-				to_chat(src,STEP_TEXT_OWNER(tail.msg_owner_help_run))
-				to_chat(tmob,STEP_TEXT_PREY(tail.msg_prey_help_run))
-			else
-				to_chat(src,"You carefully step over [tmob].")
-				to_chat(tmob,"[src] steps over you carefully!")
+				src_message = tail.msg_owner_help_run
+				tmob_message = tail.msg_prey_help_run
 
 		//Smaller person stepping under larger person
 		else if(tmob.get_effective_size() > get_effective_size() && ishuman(tmob))
+			src_message = "You run between [tmob]'s legs."
+			tmob_message = "[src] runs between your legs."
 			var/mob/living/carbon/human/H = tmob
 			if(isTaurTail(H.tail_style))
 				var/datum/sprite_accessory/tail/taur/tail = H.tail_style
-				to_chat(src,STEP_TEXT_OWNER(tail.msg_prey_stepunder))
-				to_chat(tmob,STEP_TEXT_PREY(tail.msg_owner_stepunder))
-			else
-				to_chat(src,"You run between [tmob]'s legs.")
-				to_chat(tmob,"[src] runs between your legs.")
+				src_message = tail.msg_prey_stepunder
+				tmob_message = tail.msg_owner_stepunder
+
+		to_chat(src, STEP_TEXT_OWNER(src_message))
+		to_chat(tmob, STEP_TEXT_PREY(tmob_message))
```

Comment:
These lines can end up sending `to_chat(src, "")` which isn't really the most wonderful thing.